packages/edit-site/src/store/reducer.js

```diff
@@ -183,6 +183,26 @@ export function listViewPanel( state = false, action ) {
 	return state;
 }
 
+/**
+ * Reducer to set the save view panel open or closed.
+ *
+ * @param {Object} state  Current state.
+ * @param {Object} action Dispatched action.
+ */
+export function saveViewPanel( state = false, action ) {
+	switch ( action.type ) {
```

Comment:
haven't tested yet, but should we include `SET_IS_SAVE_VIEW_OPENED` to other reducers like `listViewPanel`, etc..?

Reply:
No, I don't think so, for the listViewPanel reducer, there's a rule that says "inserter" and "list view" can't be opened at the same time, it's not the case here.